### PR TITLE
Fixed : "To merge the changes ( - Until March 31)." #196

### PR DESCRIPTION
**Pull request by daisukenishino**
# Fixes
https://github.com/OpenTouryoProject/OpenTouryo/milestone/3

### Change the structure and add license.
- https://github.com/OpenTouryoProject/OpenTouryo/commit/df48bec5382e4b9fbec85a2233622ba88db5bc94  
Replace OpenTouryo/develop/root to OpenTouryoTemplates/root_VS2015 folder
- https://github.com/OpenTouryoProject/OpenTouryo/commit/0a593d83ed322460adcb440618e5bfbf9d48888a  
Fixed OpenTouryo/README.md.
- https://github.com/OpenTouryoProject/OpenTouryo/commit/f6a9e5cbeac99fe8f7aad72a1492f13c9fa81f6f , https://github.com/OpenTouryoProject/OpenTouryo/pull/197/commits/5f3a9dcc9cd1b3db4ed27347bde39219e5da19d9  
Add a license file for templates such as project templates and class templates.

### Fixes corresponding to point out.
- Fixes to the Readme.md file.
  - https://github.com/OpenTouryoProject/OpenTouryo/pull/197/commits/c1928faf291a8069d8412e46c36f1555bb1ecbe9  
  - https://github.com/OpenTouryoProject/OpenTouryo/pull/197/commits/e07b94596b13ff146fc23c5e5b7d23226f04ffb6
- Fixes to the Readme.md and Contribute.md and LicenseForTemplate.txt file.
  - https://github.com/OpenTouryoProject/OpenTouryo/pull/197/commits/844a6e79cd915d06b71ea369b8b56be87cdf85e8

## [Fixed : "To fix encryption wrapper library and tools." #203](https://github.com/OpenTouryoProject/OpenTouryo/pull/206)
- https://github.com/OpenTouryoProject/OpenTouryo/pull/197/commits/e768ac739bd08b7efd55586c8f4b7489f36d554f
- https://github.com/OpenTouryoProject/OpenTouryo/pull/197/commits/4a33cfdab5c75c6a7ac357b7ffa1d2d34df166dd
- https://github.com/OpenTouryoProject/OpenTouryo/pull/197/commits/6a8b6651bc5deef08d8aed6b550bceffb8abbd96
- https://github.com/OpenTouryoProject/OpenTouryo/pull/197/commits/6421debfd7bf4be78a855a2a8929d85019538b66
- https://github.com/OpenTouryoProject/OpenTouryo/pull/197/commits/0a3e6418e8a63fa292fb1dbcf0ba56aaaf9bad58
- https://github.com/OpenTouryoProject/OpenTouryo/pull/197/commits/ce1a831c97fa53528b48ea8e4c40036260bc2fd7
- https://github.com/OpenTouryoProject/OpenTouryo/pull/197/commits/ecc306b0ddf7a5d681b012be66c5ea6718fb1b48
- https://github.com/OpenTouryoProject/OpenTouryo/pull/197/commits/c16b6084930b5dacec6b929b142d0189030cf8e6

## [Fixed : "Fixes of templates for prepare for Release to 02-00 version."](https://github.com/OpenTouryoProject/OpenTouryo/pull/208)
There are a lot of commits.
- https://github.com/OpenTouryoProject/OpenTouryo/pull/208/commits/e9ebb92c221afbf7c1b9d844d20aea6d87b29412
- https://github.com/OpenTouryoProject/OpenTouryo/pull/208/commits/0259c8c8301de577d3dae407fc4d03db7f1c3f59
- https://github.com/OpenTouryoProject/OpenTouryo/pull/208/commits/03acbc9aa6c435f7c11d5f473df17210280a873f
- , etc.

The rest of the information is omitted.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,11 +5,11 @@
 ## 開発／動作環境(develop/run-time environment)
```
Comment (yusukemaegawa) [inline README.md]: @daisukenishino 

Please divide this file into *English Version* and *Japanese Version*.
(Please see [this](https://github.com/OpenTouryoProject/OpenTouryo/blob/develop/root/README.md).)

File: README.md [hunk 2]
```diff
@@ -55,4 +55,4 @@ you can create a project template for the project for the appropriate Visual Stu
 以下のリポジトリを参照してください。詳しくは、各フォルダのReadmeを参照下さい。
```
Comment (yusukemaegawa) [inline README.md]: @daisukenishino 

「参照下さい」→「参照してください」

**Comment by daisukenishino** [comment]
@yusukemaegawa 

Please see https://github.com/OpenTouryoProject/OpenTouryo/pull/197/commits/c1928faf291a8069d8412e46c36f1555bb1ecbe9.

File: README.md [hunk 3]
```diff
@@ -1,58 +1,48 @@
-# Open棟梁(OpenTouryo)
-”Open棟梁”は、長年の.NETアプリケーション開発実績にて蓄積したノウハウに基づき開発した.NET用アプリケーション フレームワークです。
-(”OpenTouryo” , is an application framework for .NET which was developed using the accumulated know-how with a long track record in .NET application development.)
+# OpenTouryo
+"OpenTouryo", is an application framework for .NET which was developed using the accumulated know-how with a long track record in .NET application development.
```
Comment (yusukemaegawa) [inline README.md]: @daisukenishino 

Please revise as follows:

*OpenTouryo* is an application framework for .NET Framework which was developed using the accumulated know-how with a longstanding application development results.

File: README.md [hunk 4]
```diff
@@ -1,58 +1,48 @@
...
 
-## 開発／動作環境(develop/run-time environment)
-このリポジトリのプログラムは、サポートされる最も古い開発／実行環境を使用して開発しています。 (I am developing the program of this repository, using the oldest supported develop/run-time environment.)
-- 開発環境 (develop environment)
-    - Visual Studio 2010
-- 実行環境 (run-time environment)
-    - .NET Framework 3.5 SP1
+## Develop/Run-time environment
+The program in this repository is developed, using the oldest supported develop/run-time environment.
```
Comment (yusukemaegawa) [inline README.md]: @daisukenishino 

Please revise as follows:

The programs in this repository are developed in the following environments:

File: README.md [hunk 5]
```diff
@@ -1,58 +1,48 @@
...
-Visual Studio 2015 など、新しい開発環境を使用する場合は、[テンプレート・ベース](https://github.com/OpenTouryoProject/OpenTouryoTemplates)を参照してください。 (If you want to use the latest develop environment (e.g. Visual Studio 2015), please refer [Templates base](https://github.com/OpenTouryoProject/OpenTouryoTemplates).)
+- Develop environment
+  - Visual Studio 2015
+- Run-time environment
+  - .NET Framework 4.6
 
-## 概要(summary)
-以下のファイルを参照してください。
-(Please refer to the following files.)
+If you want to use the latest development environment (e.g. Visual Studio 2015), please refer [Templates base](https://github.com/OpenTouryoProject/OpenTouryoTemplatesV2).
```
Comment (yusukemaegawa) [inline README.md]: @daisukenishino 

Please revise as follows:

The programs in this repository are for *open source developers*.
The users who use Open Touryo in a system development project need to use [OpenTouryoTemplates repository](https://github.com/OpenTouryoProject/OpenTouryoTemplatesV2).

File: README.md [hunk 6]
```diff
@@ -1,58 +1,48 @@
...
 
+## Summary
+Please refer to the following files.
  - [Introduction.pdf](https://github.com/OpenTouryoProject/OpenTouryoDocuments/blob/master/documents/0_Introduction/ja-JP/Introduction.pdf)
  - [Introduction.pptx](https://github.com/OpenTouryoProject/OpenTouryoDocuments/blob/master/documents/0_Introduction/ja-JP/Introduction.pptx)
```
Comment (yusukemaegawa) [inline README.md]: @daisukenishino 

Please revise as follows:

[Introduction of OpenTouryo, Japanese Version (PowerPoiint)](https://github.com/OpenTouryoProject/OpenTouryoDocuments/blob/master/documents/0_Introduction/ja-JP/Introduction.pptx)

File: README.md [hunk 7]
```diff
@@ -1,58 +1,48 @@
...
  - [Introduction_EN.pdf](https://github.com/OpenTouryoProject/OpenTouryoDocuments/blob/master/documents/0_Introduction/en/Introduction_EN.pdf)
```
Comment (yusukemaegawa) [inline README.md]: @daisukenishino 

Please revise as follows:

[Introduction of OpenTouryo, English Version (PDF)](https://github.com/OpenTouryoProject/OpenTouryoDocuments/blob/master/documents/0_Introduction/en/Introduction_EN.pdf)

File: README.md [hunk 8]
```diff
@@ -1,58 +1,48 @@
...
  - [Introduction_EN.pptx](https://github.com/OpenTouryoProject/OpenTouryoDocuments/blob/master/documents/0_Introduction/en/Introduction_EN.pptx)
```
Comment (yusukemaegawa) [inline README.md]: @daisukenishino 

Please revise as follows:

[Introduction of OpenTouryo, English Version (PowerPoint)](https://github.com/OpenTouryoProject/OpenTouryoDocuments/blob/master/documents/0_Introduction/en/Introduction_EN.pptx)

File: README.md [hunk 9]
```diff
@@ -1,58 +1,48 @@
...
  - [Functional_list.xlsx](https://github.com/OpenTouryoProject/OpenTouryoDocuments/blob/master/documents/0_Introduction/ja-JP/Functional_list.xlsx)
```
Comment (yusukemaegawa) [inline README.md]: @daisukenishino 

Please revise as follows:

[Function List, Japanese Version (Excel)](https://github.com/OpenTouryoProject/OpenTouryoDocuments/blob/master/documents/0_Introduction/ja-JP/Functional_list.xlsx)

File: README.md [hunk 10]
```diff
@@ -1,58 +1,48 @@
...
 
-ドキュメント類は[OpenTouryoProject/OpenTouryoDocuments](https://github.com/OpenTouryoProject/OpenTouryoDocuments)リポジトリに格納されています。
-(documents are located in the [OpenTouryoProject/OpenTouryoDocuments](https://github.com/OpenTouryoProject/OpenTouryoDocuments) repository.)
+Documents are located in the [OpenTouryoProject/OpenTouryoDocuments](https://github.com/OpenTouryoProject/OpenTouryoDocuments) repository.
```
Comment (yusukemaegawa) [inline README.md]: @daisukenishino 

I think that this sentence is unnecessary.
(Same sentence exists in *Details* section.)

File: README.md [hunk 11]
```diff
@@ -1,58 +1,48 @@
...
 
-##詳細(details)
-以下のディレクトリを参照してください。
-(Please refer to the following directories.)
+## Details
+Please refer to the following directories.
```
Comment (yusukemaegawa) [inline README.md]: @daisukenishino 

Please revise as follows:

For more details, refer to the [OpenTouryoDocuments](https://github.com/OpenTouryoProject/OpenTouryoDocuments) repository.

File: README.md [hunk 12]
```diff
@@ -1,58 +1,48 @@
...
 
  - [documents](https://github.com/OpenTouryoProject/OpenTouryoDocuments/tree/master/documents)
```
Comment (yusukemaegawa) [inline README.md]: @daisukenishino 

Please remove this sentence.

File: README.md [hunk 13]
```diff
@@ -1,58 +1,48 @@
...
 
-ドキュメント類は[OpenTouryoProject/OpenTouryoDocuments](https://github.com/OpenTouryoProject/OpenTouryoDocuments)リポジトリに格納されています。
-(documents are located in the [OpenTouryoProject/OpenTouryoDocuments](https://github.com/OpenTouryoProject/OpenTouryoDocuments) repository.)
+Documents are located in the [OpenTouryoProject/OpenTouryoDocuments](https://github.com/OpenTouryoProject/OpenTouryoDocuments) repository.
```
Comment (yusukemaegawa) [inline README.md]: @daisukenishino 

Please remove this sentence.

File: README.md [hunk 14]
```diff
@@ -1,58 +1,48 @@
...
 
-##内容物(contents)
+## Contents
 
-###ディレクトリ(directory)
+### Directory
 
 #### [/license/](https://github.com/OpenTouryoProject/OpenTouryo/tree/master/license)
-ライセンス(license)
+License
```
Comment (yusukemaegawa) [inline README.md]: @daisukenishino 

Please revise as follows:

License files are stored in this directory.

File: README.md [hunk 15]
```diff
@@ -1,58 +1,48 @@
...
 
 #### [/root/](https://github.com/OpenTouryoProject/OpenTouryo/tree/master/root)
-プログラム
+Program
```
Comment (yusukemaegawa) [inline README.md]: @daisukenishino 

Please revise as follows:

Programs, configuration files, sql files, and more are stored in this directory.

File: README.md [hunk 16]
```diff
@@ -1,58 +1,48 @@
...
 
-##テンプレート・ベース(Templates base)
-テンプレート・ベースに同梱されるサンプルはOpen棟梁の評価に利用できます。
-また、テンプレート・ベースをチュートリアルの内容に従いカスタマイズすることで
-当該Visual Studioバージョンの案件向けプロジェクト・テンプレートを作成できます。 
-
-(Samples that are shipped with the templates base can be used to evaluate the OpenTouryo. 
+## Templates base
+Samples that are shipped with the templates base can be used to evaluate the OpenTouryo.
```
Comment (yusukemaegawa) [inline README.md]: @daisukenishino 

Please revise as follows:

*Open Touryo Template Base* is the *foundation* of the development infrastructure (project template) of the programs using Open Touryo.
The samples included in *Open Touryo Template Base* can be used to evaluate Open Touryo.

File: README.md [hunk 17]
```diff
@@ -1,58 +1,48 @@
...
 
-##テンプレート・ベース(Templates base)
-テンプレート・ベースに同梱されるサンプルはOpen棟梁の評価に利用できます。
-また、テンプレート・ベースをチュートリアルの内容に従いカスタマイズすることで
-当該Visual Studioバージョンの案件向けプロジェクト・テンプレートを作成できます。 
-
-(Samples that are shipped with the templates base can be used to evaluate the OpenTouryo. 
+## Templates base
+Samples that are shipped with the templates base can be used to evaluate the OpenTouryo. 
 Further, by using templates base customized according to the contents of the tutorial,
```
Comment (yusukemaegawa) [inline README.md]: @daisukenishino 

Please revise as follows:

When the mismatch is generated between *the features of Open Touryo* and *the requirements of the development project*, the customizing template base is useful for resolving the mismatch.  
Refer to the [tutorial document](https://github.com/OpenTouryoProject/OpenTouryoDocuments/blob/master/documents/2_Tutorial/ja-JP/Tutorial_Template_development.doc) about the customizing method of template base.

File: README.md [hunk 18]
```diff
@@ -1,58 +1,48 @@
...
-(Samples that are shipped with the templates base can be used to evaluate the OpenTouryo. 
+## Templates base
+Samples that are shipped with the templates base can be used to evaluate the OpenTouryo. 
 Further, by using templates base customized according to the contents of the tutorial, 
-you can create a project template for the project for the appropriate Visual Studio version.)
+you can create a project template for the project for the appropriate Visual Studio version.
 
-以下のリポジトリを参照してください。詳しくは、各フォルダのReadmeを参照下さい。
-(Please refer to the following repository. For more information, please refer to the Readme for each folder.)
+Please refer to the following repository. For more information, please refer to the Readme for each folder.
```
Comment (yusukemaegawa) [inline README.md]: @daisukenishino 

Please revise as follows:

For more information, please refer to the Readme files in the following repository.

File: Readme.ja.md [hunk 19]
```diff
@@ -0,0 +1,49 @@
+# Open棟梁
+”Open棟梁”は、長年の.NETアプリケーション開発実績にて蓄積したノウハウに基づき開発した.NET用アプリケーション フレームワークです。
+
+## 開発 / 動作環境
+このリポジトリのプログラムは、サポートされる最も古い開発／実行環境を使用して開発しています。
```
Comment (yusukemaegawa) [inline Readme.ja.md]: @daisukenishino 

Please revise as follows:

このリポジトリのプログラムは、以下の開発／実行環境を使用して開発しています。

File: Readme.ja.md [hunk 20]
```diff
@@ -0,0 +1,49 @@
...
+
+- 開発環境
+  - Visual Studio 2015
+- 実行環境
+  - .NET Framework 4.6
+
+新しい開発環境を使用する場合は、[テンプレート・ベース](https://github.com/OpenTouryoProject/OpenTouryoTemplatesV2)を参照してください。
```
Comment (yusukemaegawa) [inline Readme.ja.md]: @daisukenishino 

Please revise as follows:

このリポジトリのプログラムは、OSS 開発者向けのものです。
システム開発プロジェクトで Open 棟梁を使用される方は、[OpenTouryoTemplates リポジトリ](https://github.com/OpenTouryoProject/OpenTouryoTemplates)を使用してください。

File: Readme.ja.md [hunk 21]
```diff
@@ -0,0 +1,49 @@
...
+  - Visual Studio 2015
+- 実行環境
+  - .NET Framework 4.6
+
+新しい開発環境を使用する場合は、[テンプレート・ベース](https://github.com/OpenTouryoProject/OpenTouryoTemplatesV2)を参照してください。 
+
+## 概要
+以下のファイルを参照してください。
+
+ - [Introduction.pdf](https://github.com/OpenTouryoProject/OpenTouryoDocuments/blob/master/documents/0_Introduction/ja-JP/Introduction.pdf)
```
Comment (yusukemaegawa) [inline Readme.ja.md]: @daisukenishino

Please revise as follows:

- [概要資料 日本語版 (PDF)](https://github.com/OpenTouryoProject/OpenTouryoDocuments/blob/master/documents/0_Introduction/ja-JP/Introduction.pdf)

File: Readme.ja.md [hunk 22]
```diff
@@ -0,0 +1,49 @@
...
+ - [Introduction.pptx](https://github.com/OpenTouryoProject/OpenTouryoDocuments/blob/master/documents/0_Introduction/ja-JP/Introduction.pptx)
```
Comment (yusukemaegawa) [inline Readme.ja.md]: @daisukenishino

Please revise as follows:

- [概要資料 日本語版 (PowerPoint)](https://github.com/OpenTouryoProject/OpenTouryoDocuments/blob/master/documents/0_Introduction/ja-JP/Introduction.pptx)

File: Readme.ja.md [hunk 23]
```diff
@@ -0,0 +1,49 @@
...
+ - [Introduction_EN.pdf](https://github.com/OpenTouryoProject/OpenTouryoDocuments/blob/master/documents/0_Introduction/en/Introduction_EN.pdf)
```
Comment (yusukemaegawa) [inline Readme.ja.md]: @daisukenishino

Please revise as follows:

- [概要資料 英語版 (PDF)](https://github.com/OpenTouryoProject/OpenTouryoDocuments/blob/master/documents/0_Introduction/en/Introduction_EN.pdf)

File: Readme.ja.md [hunk 24]
```diff
@@ -0,0 +1,49 @@
...
+ - [Introduction_EN.pptx](https://github.com/OpenTouryoProject/OpenTouryoDocuments/blob/master/documents/0_Introduction/en/Introduction_EN.pptx)
```
Comment (yusukemaegawa) [inline Readme.ja.md]: @daisukenishino

Please revise as follows:

- [概要資料 英語版 (PowerPoint)](https://github.com/OpenTouryoProject/OpenTouryoDocuments/blob/master/documents/0_Introduction/en/Introduction_EN.pptx)

File: Readme.ja.md [hunk 25]
```diff
@@ -0,0 +1,49 @@
...
+ - [Functional_list.xlsx](https://github.com/OpenTouryoProject/OpenTouryoDocuments/blob/master/documents/0_Introduction/ja-JP/Functional_list.xlsx)
```
Comment (yusukemaegawa) [inline Readme.ja.md]: @daisukenishino

Please revise as follows:

- [機能一覧 日本語版 (Excel)](https://github.com/OpenTouryoProject/OpenTouryoDocuments/blob/master/documents/0_Introduction/ja-JP/Functional_list.xlsx)

File: Readme.ja.md [hunk 26]
```diff
@@ -0,0 +1,49 @@
...
+
+ドキュメント類は[OpenTouryoProject/OpenTouryoDocuments](https://github.com/OpenTouryoProject/OpenTouryoDocuments)リポジトリに格納されています。
```
Comment (yusukemaegawa) [inline Readme.ja.md]: @daisukenishino

I think that this sentence is unnecessary.
(Same sentence exists in 詳細 section.)

File: Readme.ja.md [hunk 27]
```diff
@@ -0,0 +1,49 @@
...
+
+## 詳細
+以下のディレクトリを参照してください。
```
Comment (yusukemaegawa) [inline Readme.ja.md]: @daisukenishino

Please revise as follows:

Open 棟梁のドキュメントは、[OpenTouryoDocuments リポジトリ](https://github.com/OpenTouryoProject/OpenTouryoDocuments)にあります。
詳細は、[OpenTouryoDocuments リポジトリ](https://github.com/OpenTouryoProject/OpenTouryoDocuments)のドキュメントを参照してください。

File: Readme.ja.md [hunk 28]
```diff
@@ -0,0 +1,49 @@
...
+
+ - [documents](https://github.com/OpenTouryoProject/OpenTouryoDocuments/tree/master/documents)
```
Comment (yusukemaegawa) [inline Readme.ja.md]: @daisukenishino

Please remove this sentence.

File: Readme.ja.md [hunk 29]
```diff
@@ -0,0 +1,49 @@
...
+
+## 内容物
+
+### ディレクトリ
+
+#### [/license/](https://github.com/OpenTouryoProject/OpenTouryo/tree/master/license)
+ライセンス
+
+#### [/root/](https://github.com/OpenTouryoProject/OpenTouryo/tree/master/root)
+プログラム
```
Comment (yusukemaegawa) [inline Readme.ja.md]: @daisukenishino

Please revise as follows:

このディレクトリには、プログラム、設定ファイル、SQL ファイルなどが格納されています。

File: Readme.ja.md [hunk 30]
```diff
@@ -0,0 +1,49 @@
...
+
+## テンプレート・ベース
+テンプレート・ベースに同梱されるサンプルはOpen棟梁の評価に利用できます。
```
Comment (yusukemaegawa) [inline Readme.ja.md]: @daisukenishino

Please revise as follows:

「Open 棟梁テンプレート・ベース」とは、Open 棟梁を使用したシステムの開発基盤 (プロジェクトテンプレート) を作るための元になるものです。
Open 棟梁テンプレート・ベースに含まれるサンプルは、Open 棟梁の評価に利用できます。

File: Readme.ja.md [hunk 31]
```diff
@@ -0,0 +1,49 @@
...
+また、テンプレート・ベースをチュートリアルの内容に従いカスタマイズすることで
+当該Visual Studioバージョンの案件向けプロジェクト・テンプレートを作成できます。 
+
+以下のリポジトリを参照してください。詳しくは、各フォルダのReadmeを参照してください。
```
Comment (yusukemaegawa) [inline Readme.ja.md]: @daisukenishino 

Please revise as follows:

詳細は、以下リポジトリの Readme ファイルを参照してください。

File: root/README.md [hunk 32]
```diff
@@ -1,60 +1,18 @@
...
-- OpenTouryo repository (*Current repository*)
-    - Summary  
-    The programs in this repository are the *matrix* of OpenTouryoTemplates repository.  
-    (First, the features of Open Touryo are implemented in this repository. And then, the features are introduced into OpenTouryoTemplates repository.)
-    - Intended User  
-    *Open source developers*.
-
-Therefore, the users who use Open Touryo in a system development project need to use [OpenTouryoTemplates repository](https://github.com/OpenTouryoProject/OpenTouryoTemplates).  
-The following contents are the usage of Open Touryo for *open source developers*.
+# Open Touryo Template Base for Visual Studio 2015
```
Comment (yusukemaegawa) [inline root/README.md]: @daisukenishino 

This repository is *NOT* template base.
Please do not replace this file with Readme file of template base.

File: root/Readme.ja.md [hunk 33]
```diff
@@ -1,64 +1,21 @@
...
-    SQL Server を既定の DBMS とし、Visual Studio のバージョンごとにフォルダが分かれている
-    - 想定利用者  
-    Open 棟梁を使用してアプリケーションを開発する人
-- OpenTouryo リポジトリ (**現在のリポジトリ**)
-    - 概要  
-    OpenTouryoTemplates リポジトリの**母体**となるもの  
-    (Open 棟梁の各機能は、まず OpenTouryo リポジトリで開発され、その後 OpenTouryoTemplates リポジトリに取り込まれる)
-    - 想定利用者  
-    OSS 開発者
+# Open 棟梁テンプレート・ベース (Visual Studio 2015 用)
```
Comment (yusukemaegawa) [inline root/Readme.ja.md]: @daisukenishino

This repository is *NOT* template base.
Please do not replace this file with Readme file of template base.

File: root/programs/C#/Frameworks/Infrastructure/Business/Business/MyApsBaseLogic.cs [hunk 34]
```diff
@@ -221,90 +221,97 @@ protected override void UOC_DoAction(BaseParameterValue parameterValue, ref Base
...
-                }
-                else if (parameterValue.ActionType.Split('%')[0] == "ODP")
-                {
-                    // Oracle / ODP.NET用のDamを生成
-                    damABT = new DamOraOdp();
-                    damAMT = new DamOraOdp();
+                //    // 接続文字列をロード
+                //    connstring = GetConfigParameter.GetConnectionString("ConnectionString_OLE");
+                //}
+                //else if (parameterValue.ActionType.Split('%')[0] == "ODB")
```
Comment (yusukemaegawa) [inline root/programs/C#/Frameworks/Infrastructure/Business/Business/MyApsBaseLogic.cs]: @daisukenishino 

This program is only enabled for SQL Server.
To enable this program for other databases, uncomment these lines.

**Comment by yusukemaegawa** [comment]
@daisukenishino 

Has [programs\C#\Tests folder](https://github.com/OpenTouryoProject/OpenTouryo/tree/old_master_v1/root/programs/C%23/Tests) gone?

**Comment by yusukemaegawa** [comment]
@daisukenishino

Please divide CONTRIBUTING.md into English Version and Japanese Version.

**Comment by yusukemaegawa** [comment]
@daisukenishino 

Is [this file](https://github.com/OpenTouryoProject/OpenTouryo/blob/develop/license/README.txt) necessary?

**Comment by daisukenishino** [comment]
- https://github.com/OpenTouryoProject/OpenTouryo/pull/197#pullrequestreview-15439471  
As for this, I think that I will do the work after copying this to Template V2.
- https://github.com/OpenTouryoProject/OpenTouryo/pull/197#issuecomment-270829867  
  - Because this folder was not included in the root_VS2016 folder.
  - I will plan to recover this in the future.
  - I think that it is necessary to make changes to the scope of test automation as well.
- https://github.com/OpenTouryoProject/OpenTouryo/pull/197#issuecomment-270830946  
I see, I will divide this file.
- https://github.com/OpenTouryoProject/OpenTouryo/pull/197#issuecomment-270831037  
I also think that it is unnecessary and I will delete it.

**Comment by daisukenishino** [comment]
@yusukemaegawa 

Please tell me the followings.
- https://github.com/OpenTouryoProject/OpenTouryo/pull/197#pullrequestreview-15438758
- https://github.com/OpenTouryoProject/OpenTouryo/pull/197#pullrequestreview-15438852

Should these be reverted to the following?
- https://github.com/OpenTouryoProject/OpenTouryo/blob/master/root/README.md
- https://github.com/OpenTouryoProject/OpenTouryo/blob/master/root/Readme.ja.md

Also, I think that it is necessary to change the VS version as follows.
- Visual Studio 2010 --->Visual Studio 2015

File: README.md [hunk 35]
```diff
@@ -1,58 +1,45 @@
-# Open棟梁(OpenTouryo)
-”Open棟梁”は、長年の.NETアプリケーション開発実績にて蓄積したノウハウに基づき開発した.NET用アプリケーション フレームワークです。
-(”OpenTouryo” , is an application framework for .NET which was developed using the accumulated know-how with a long track record in .NET application development.)
+# OpenTouryo
+*Open Touryo* is an application framework for .NET Framework which was developed using the accumulated know-how with a longstanding application development results.
```
Comment (yusukemaegawa) [inline README.md]: @daisukenishino 

Please add link to Japanese version.
See [readme file in OpenTouryoTempates repository](https://github.com/OpenTouryoProject/OpenTouryoTemplates/blob/master/root_VS2015/Readme.md).
```md
Click [here](Readme.ja.md) for Japanese version of this file.
```

File: Readme.ja.md [hunk 36]
```diff
@@ -0,0 +1,45 @@
+# Open棟梁
+”Open棟梁”は、長年の.NETアプリケーション開発実績にて蓄積したノウハウに基づき開発した.NET用アプリケーション フレームワークです。
+
```
Comment (yusukemaegawa) [inline Readme.ja.md]: @daisukenishino 

Please add link to English version.
See [readme file in OpenTouryoTempates repository](https://github.com/OpenTouryoProject/OpenTouryoTemplates/blob/master/root_VS2015/Readme.ja.md).
```md
このファイルの英語版は[こちら](Readme.md)から。
```

**Comment by daisukenishino** [comment]
@yusukemaegawa 

After the correction as the material is completed, I ask you the merge.
Please wait for the merge of this pull-request until then.

**Comment by daisukenishino** [comment]
@yusukemaegawa

This work is completed. Please check the work.
But, Please perform this merge after March 31st.

**Pull request merged**